packages/shared/README.md

```
@@ -0,0 +1,98 @@
+
+## Contents
+
+### Constants
+
+- **Time constants**: `ONE_SECOND` (1000ms), `HALF_SECOND` (500ms), `TTI_BUFFER_MS` (1000ms)
+- **Size constants**: `BYTES_TO_KB` (1024), `KILOBYTE` (1024)
+- **Percentage constants**: `PERCENTAGE_MULTIPLIER` (100), `PERCENTAGE_DIVISOR` (100)
+
+### Utilities
+
+#### Time Formatting
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)